### PR TITLE
add mission vision value page

### PR DESCRIPTION
**Pull request by magdmartin**
This PR adds the results of the work to clarify OpenRefine's Mission, Vision, and Values following the project completed in 2024. see the [related discussion on the forum](https://forum.openrefine.org/t/feedback-request-on-openrefines-draft-vision-mission-values/1619/8). 

The integration in the website was previously discussed in #291 and on the [forum](https://forum.openrefine.org/t/requesting-feedback-documenting-openrefine-community-handbooks/1224/2)

### CODE REVIEW

File: src/pages/mission_vision.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: OpenRefine Mission and Vision
+description: OpenRefine Mission and Vision
+hide_table_of_contents: true
+---
+
+## OpenRefine Vision
+ 
+A more informed world where working with data is easy and engaging
```
Comment (Ainali) [inline src/pages/mission_vision.md:9]:
```suggestion
A more informed world where working with data is easy and engaging.
```

File: src/pages/mission_vision.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+### Approachable
+
+We ensure that OpenRefine is inviting and supportive of all participants by providing various on-ramps to engagement with the tool and community, including but not limited to clear documentation, thoughtful mentorship and training, and by prioritizing ease-of-use and participation across backgrounds and skill-levels. We expect everyone to engage with kindness, warmth, patience, and an unwavering commitment to mutual support.
+
+### Transparent & Open
+
+We create pathways for and encourage clear communication and knowledge-sharing, ranging from regularly-updated documentation to consistent public participation in community forums. We believe that transparent practices are essential to a collaborative community where all participants – including those with limited time and experience – are informed and feel empowered to engage. The tool mirrors this commitment to transparency through prioritizing the use of open and commons-based data, and the privacy of all users
```
Comment (Ainali) [inline src/pages/mission_vision.md:42]:
```suggestion
We create pathways for and encourage clear communication and knowledge-sharing, ranging from regularly-updated documentation to consistent public participation in community forums. We believe that transparent practices are essential to a collaborative community where all participants – including those with limited time and experience – are informed and feel empowered to engage. The tool mirrors this commitment to transparency through prioritizing the use of open and commons-based data, and the privacy of all users.
```

File: src/pages/mission_vision.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### Curious
+
+We believe in the importance of a learning mindset, where everyone leads with questions and engages with humility. We recognize that we have a lot to learn from one another, and seek opportunities to foster our mutual growth. This includes being thoughtful in how we give and receive feedback, consistently examining our own biases, and providing proactive support to new learners in our community. For the tool itself, we operate with humility and listen to end-users to prioritize their needs and plan for features and iterations that optimize their use of the tool.
+
+### Community-driven
+
+We seek to facilitate a sense of community among users and contributors, and to prioritize the perspectives of community members in key decisions. We believe that our users and contributors dene OpenRefine, and should collaboratively drive our work forward – not as individuals but as members of a supportive and vibrant community. This is reected in open communication about the future of the community and tool, a sense of agency among community members, and in democratic decision-making around strategic and technical priorities.
```
Comment (Ainali) [inline src/pages/mission_vision.md:50]:
```suggestion
We seek to facilitate a sense of community among users and contributors, and to prioritize the perspectives of community members in key decisions. We believe that our users and contributors define OpenRefine, and should collaboratively drive our work forward – not as individuals but as members of a supportive and vibrant community. This is reflected in open communication about the future of the community and tool, a sense of agency among community members, and in democratic decision-making around strategic and technical priorities.
```

**Comment by Ainali** [comment]
Great! A couple of small typos, and then I wonder if we should remove the unusual amount of blank lines.

**Comment by magdmartin** [comment]
thanks @Ainali

**Comment by magdmartin** [comment]
I will merge this tomorrow if there is no objection.

**Pull request merged**